pkg/chains/signing.go

```diff
@@ -44,6 +46,9 @@ type ObjectSigner struct {
 	Backends          map[string]storage.Backend
 	SecretPath        string
 	Pipelineclientset versioned.Interface
+	// Metrics Recorder config
+	PrMetrics *pipelinerunmetrics.Recorder
```

Comment:
We can have an interface here: recorder. 

This interface has two functions: RecordSignedCountMetrics and RecordPayloadCountMetrics.

Reply:
Updated